src/live_restore/live_restore.h

```diff
@@ -8,22 +8,36 @@
 
 #pragma once
 
+/*
+ * Live restore state reported to the application so it knows when to terminate live restore.
+ *
+ * !!! Applications don't have access to these macros and instead checks them by value. i.e. to
+ * know live restore has completed the application reads the stat and checks for the value 2. Do
+ * not change these values without updating the relevant code in the application layer.
+ */
 #define WT_LIVE_RESTORE_INIT 0x0
 #define WT_LIVE_RESTORE_IN_PROGRESS 0x1
 #define WT_LIVE_RESTORE_COMPLETE 0x2
```

Comment:
If an application looks at state values, then these values must be part of `src/include/wiredtiger.in`. You can look at `WT_BACKUP_FILE` and `WT_BACKUP_RANGE` for exposing values to applications. The application should just be able to look for `WT_LIVE_RESTORE_COMPLETE` no matter if its value needs to change in the future. Those values can/should move to `wiredtiger.in` and be available to applications.

Reply:
Thanks! This is something I was looking for earlier but wasn't sure how to implement.
Moved in https://github.com/wiredtiger/wiredtiger/pull/11590/commits/2ba6a595f26302ab0d389cef28a2aadc3c202044
CC @Sean04. Once this is merged we can update the server-side code to check the enum instead of a raw `== 2`